### ISSUE TITLE
eating blocks[Bug] ...

### ISSUE THREAD

**Issue by FeryFaithless** [SuperMartijn642/Rechiseled#79]
- Minecraft, ...
- Rechiseled, ...
- Fusion, ...

i use this mod in 1.16.5 forge and i have many mods.near 300 n all b ok but tis Rechiseled can make just one block.surry for poor english.its voide all my other blocks and makes just once.ow we can do for fixing!tnx

**Comment by SuperMartijn642** [comment]
I don't understand your message at all.

Also please fill in the template. I have no clue what Minecraft version or what version of Rechiseled you are using?

**Comment by FeryFaithless** [comment]
i think i found t problem .uh,its: rechiseled-1.1.5a-forge-mc1.16.jar  and latest forge version for forge.
and there was a problem with optifine.when i turned off connected texture now evrything is ok.
next time (i cnt speak well in english.and must be,) next time if had tis prob again,i try for record screen 10 second and send ,its better way.

***The summary is that last night, before the problem was solved, when I put anumber of blocks (any number) in Chisel, it would become one. The numbers would disappear and the output would be1block.and even If there were many blocks in my pocket, for example, 5 stacks, and I clicked on chiseled all, they would all become one just.where goes other blocks,i dont know ((: ***

i just turned off optifine connected texture in options and restart again the game,and fixed after one month and many trys,or maybe few days ago turned off some mods have overlaes and entity clum shadows changing values in options or off/delete this 2 mod.But maybe no,and its just before optifine,i think ,its was just for this.Anyway dear, good luck and ,
i wish t best for you and u r our pride ,god bless you,.
